lmdk/README.md

```diff
@@ -0,0 +1,15 @@
+# Loadable Modules Dev Kit
+
+***TODO: add link to the documentation repo!***
```

Comment:
@serhiy-katsyuba-intel  I think this TODO and the TODO in commit is confusing a bit. Is this PR intended for merging now, or do you plan to work on this still before merging?

Reply:
I've made an introduction part in sof-docs https://github.com/thesofproject/sof-docs/pull/463, it is lacking architecture part, but I'm going to fill it up after May break.